scripts/release/asset_publish.py

```diff
@@ -301,9 +341,24 @@ def _str2bool(v: str) -> bool:
                 except Exception as e:
                     logger.log_error(f"Exception in loading model config: {str(e)}")
                     failure_list.append(asset)
-
+            elif asset.type == assets.AssetType.DATA:
+                try:
+                    data_config = asset.extra_config_as_object()
+                    with TemporaryDirectory() as tempdir:
+                        result = data_prepare(data_config, asset.spec_with_path, Path(tempdir))
+                        if result:
+                            # Assemble Command
+                            cmd = assemble_command(
+                                asset.type.value, str(asset.spec_with_path),
+                                registry_name, asset.version, resource_group, workspace, debug_mode)
+                            # Run command
+                            run_command(cmd, failure_list, debug_mode)
+                except Exception as e:
+                    logger.log_error(f"Exception in publishing data asset: {str(e)}")
```

Comment:
F-strings don't require extra `str` conversion.
```suggestion
                    logger.log_error(f"Exception in publishing data asset: {e}")
```